cvat/apps/reid/static/reid/js/enginePlugin.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/* global showMessage userConfirm */
+
+
+document.addEventListener('DOMContentLoaded', () => {
+    function run(reidButton, tresholdInput, distanceInput) {
```

Comment:
I see a lot of typos: treshold --> threshold! (in UI as well)

Reply:
Done